src/lzma.cpp

```diff
@@ -75,7 +82,7 @@ void lzma_error::check BOOST_PREVENT_MACRO_SUBSTITUTION(int error)
 namespace detail {
 
 lzma_base::lzma_base()
-    : stream_(new lzma_stream), level(lzma::default_compression)
+    : stream_(new lzma_stream), level_(lzma::default_compression), threads_(1)
```

Comment:
That would work, with just a change where you commented. I was trying to avoid surprising the users by changing the default behavior. If you think that `0` is the right answer, I can make that change.

Reply:
Correction: a change here would be ignored. `threads_` should always be overwritten by whatever is passed in `lzma_params`, even if that's the default constructed version of `lzma_params`. I would change both to 0.